Master build is broken: error NU1603

No code has changed but the build is broken, the master Castle Core build isn't broken (even after I reran it too).

```
NUnit.Engine.NUnitEngineException: The path specified in --result src\Castle.Windsor.Tests\bin\Release\net45\TestResult_Windsor.xml could not be written to ---> System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\projects\windsor\src\Castle.Windsor.Tests\bin\Release\net45\TestResult_Windsor.xml'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at NUnit.Engine.Services.NUnit3XmlResultWriter.WriteResultFile(XmlNode resultNode, String outputPath)
   at NUnit.ConsoleRunner.ConsoleRunner.RunTests(TestPackage package, TestFilter filter)
   --- End of inner exception stack trace ---
   at NUnit.ConsoleRunner.ConsoleRunner.RunTests(TestPackage package, TestFilter filter)
   at NUnit.ConsoleRunner.Program.Main(String[] args)
```

The version of NUnit hadn't changed, it is actually caused by the assembly failing compilation because of this warning (converted to an error by warnings as errors).

> C:\projects\windsor\src\Castle.Windsor\Castle.Windsor.csproj : error NU1603: Castle.Core 4.0.0 depends on System.ComponentModel.TypeConverter (>= 4.0.1) but System.ComponentModel.TypeConverter 4.0.1 was not found. An approximate best match of System.ComponentModel.TypeConverter 4.1.0 was resolved.

I guess the .NET CLI tooling has been updated either by AppVeyor or our build just pulls the latest.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,10 @@
 # Castle Windsor Changelog
 
+## Unreleased
+
+Bugfixes:
+- Fix warnings regarding non-existant System.ComponentModel.TypeConverter NuGet package by updating minimum Castle Core version to 4.1.0 (#321)
+
 ## 4.0.0 (2017-07-12)
 
 Breaking Changes:

--- a/src/Castle.Facilities.WcfIntegration.Demo/Castle.Facilities.WcfIntegration.Demo.csproj
+++ b/src/Castle.Facilities.WcfIntegration.Demo/Castle.Facilities.WcfIntegration.Demo.csproj
@@ -11,7 +11,7 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Castle.Core-log4net" Version="4.0.0" />
+		<PackageReference Include="Castle.Core-log4net" Version="4.1.0" />
 	</ItemGroup>
 
 	<ItemGroup>

--- a/src/Castle.Facilities.WcfIntegration.Tests/Castle.Facilities.WcfIntegration.Tests.csproj
+++ b/src/Castle.Facilities.WcfIntegration.Tests/Castle.Facilities.WcfIntegration.Tests.csproj
@@ -12,8 +12,8 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Castle.Core" Version="4.0.0" />
-		<PackageReference Include="Castle.Core-log4net" Version="4.0.0" />
+		<PackageReference Include="Castle.Core" Version="4.1.0" />
+		<PackageReference Include="Castle.Core-log4net" Version="4.1.0" />
 		<PackageReference Include="NUnit" Version="3.6.1" />
 		<PackageReference Include="NUnit.Console" Version="3.6.1" />
 	</ItemGroup>

--- a/src/Castle.Windsor.Tests/Castle.Windsor.Tests.csproj
+++ b/src/Castle.Windsor.Tests/Castle.Windsor.Tests.csproj
@@ -58,8 +58,8 @@
 	</ItemGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)'=='net45'">
-		<PackageReference Include="Castle.Core-log4net" Version="4.0.0" />
-		<PackageReference Include="Castle.Core-NLog" Version="4.0.0" />
+		<PackageReference Include="Castle.Core-log4net" Version="4.1.0" />
+		<PackageReference Include="Castle.Core-NLog" Version="4.1.0" />
 	</ItemGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)'=='netcoreapp1.0'">

--- a/src/Castle.Windsor/Castle.Windsor.csproj
+++ b/src/Castle.Windsor/Castle.Windsor.csproj
@@ -23,7 +23,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Castle.Core" Version="4.0.0" />
+		<PackageReference Include="Castle.Core" Version="4.1.0" />
 	</ItemGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)'=='net45'">

